src/utils/cairoTypeSystem.ts

```diff
@@ -248,6 +248,29 @@ export class CairoTuple extends CairoType {
   }
 }
 
+export class CairoStaticArray extends CairoType {
+  constructor(public type: CairoType, public size: number) {
+    super();
+  }
+  get fullStringRepresentation(): string {
+    return `[Tuple]${`(${this.type.fullStringRepresentation})`.repeat(this.size)}`;
```

Comment:
I think the keyword `Tuple` in `fullStringRepresentation` should be changed to something unique different than `CairoTuple`. Perhaps something like `StaticArray` or `SameTypeTuple`.

Do we still need `CairoTuple`?

Reply:
Also make the string readable, a uint8[2] will be smth like "[Tuple]feltfelt" which may hurt readability. Since we know there is only one type, for the previous example you could print instead "[Tuple][2][felt]" meaning a static array of size  two where each element is a felt